src/containers/PharmacyList/PharmacyList.tsx

```diff
@@ -13,16 +13,24 @@ interface Props {
 }
 
 export function PharmacyList(props: Props) {
+  const postalCodeSearch = props.postalCode
```

Comment:
Any value in applying the postal code regex to the `postalCode` prop to ensure conformance? Humans could mistype a URL and supply an incorrect postal code by accident.